tox.ini

```diff
@@ -48,24 +47,14 @@ commands = python -m flake8 --show-source tox setup.py {posargs}
 max-complexity = 22
 max-line-length = 99
 
-[testenv:py26-bare]
-description = invoke the tox help message under Python 2.6
-install_command = pip install {opts} {packages}
-list_dependencies_command = pip freeze
-deps =
-commands = tox -h
-
-[testenv:py26]
-install_command = pip install {opts} {packages}
-list_dependencies_command = pip freeze
-
 [testenv:X]
 description = print the positional arguments passed in with echo
 commands = echo {posargs}
 
 [testenv:coverage]
 description = [run locally after tests]: combine coverage data and create reports
 skip_install = True
+changedir={toxworkdir}
```

Comment:
this is unrelated to the changes here (and it seems breaking windows) -- did you accidentally commit this over another WIP branch?

Reply:
no accident, will take a look